docker/test/fasttest/run.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -x -e
+
+ls -la
+
+git clone https://github.com/ClickHouse/ClickHouse.git | ts '%Y-%m-%d %H:%M:%S' | tee /test_output/clone_log.txt
+cd ClickHouse
+CLICKHOUSE_DIR=`pwd`
+
+
+if [ "$PULL_REQUEST_NUMBER" != "0" ]; then
+    if git fetch origin "+refs/pull/$PULL_REQUEST_NUMBER/merge"; then
+        git checkout FETCH_HEAD
+        echo 'Clonned merge head'
+    else
+        git fetch
+        git checkout $COMMIT_SHA
+        echo 'Checked out to commit'
+    fi
+else
+    if [ "$COMMIT_SHA" != "" ]; then
+        git checkout $COMMIT_SHA
+    fi
+fi
+
+SUBMODULES_TO_UPDATE="contrib/boost contrib/zlib-ng contrib/libxml2 contrib/poco contrib/libunwind contrib/ryu contrib/fmtlib contrib/base64 contrib/cctz contrib/libcpuid contrib/replxx contrib/double-conversion contrib/libcxx contrib/libcxxabi contrib/libc-headers contrib/lz4 contrib/zstd contrib/fastops contrib/rapidjson contrib/re2 contrib/sparsehash-c11"
+
+git submodule update --init --recursive $SUBMODULES_TO_UPDATE | ts '%Y-%m-%d %H:%M:%S' | tee /test_output/submodule_log.txt
+
+export CMAKE_LIBS_CONFIG="-DENABLE_RDKAFKA=0 -DENABLE_S3=0 -DUSE_SENTRY=0 -DENABLE_AMQPCPP=0 -DENABLE_HDFS=0 -DENABLE_MYSQL=0 -DENABLE_GRPC=0 -DENABLE_CURL=0 -DENABLE_JEMALLOC=0 -DENABLE_CAPNP=0 -DENABLE_RDKAFKA=0 -DENABLE_ICU=0 -DENABLE_POCO_NETSSL=0 -DENABLE_ODBC=0 -DENABLE_MYSQL=0 -DENABLE_SSL=0 -DENABLE_POCO_NETSSL=0 -DENABLE_CASSANDRA=0 -DENABLE_PARQUET=0 -DENABLE_AVRO=0 -DENABLE_EMBEDDED_COMPILER=0 -DENABLE_PROTOBUF=0 -DENABLE_H3=0 -DENABLE_TESTS=0 -DENABLE_UTILS=0 -DENABLE_ODBC=0 -DENABLE_GSASL_LIBRARY=0 -DENABLE_THINLTO=0 -DENABLE_MSGPACK=0 -DENABLE_LDAP=0 -DUSE_SNAPPY=0 -DENABLE_HYPERSCAN=0"
```

Comment:
Maybe an allow-list with `ENABLE_LIBRARIES=0` and `ENABLE_MY_FAVORITE_LIBRARY=1`? This is going to break when we add new ones.